jacob-david/__test__/transform.test.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const transform = require('../lib/transform');
+
+describe('transform-test.js', () => {
+    test('the transformation we applied should be returned if there are no errors', (done) {
+
+    transform.readFile((error,data) => {
```

Comment:
this test is failing because:
```
    TypeError: transform.readFile is not a function
      
      at Object.done (__test__/transform.test.js:8:15)
```

the readFile function doesn't exist in transform.js, it's in index.js